physical/consul.go

```diff
@@ -79,6 +76,7 @@ func newConsulBackend(conf map[string]string, logger *log.Logger) (Backend, erro
 	if !ok {
 		path = "vault/"
 	}
+	logger.Printf("[DEBUG]: consul: config path set to %v", path)
```

Comment:
Nitpick, the `[DEBUG]` should not be followed by a colon, applies to all log lines.